client/src/app/game-details/game-details.component.ts

```diff
@@ -35,4 +39,23 @@ export class GameDetailsComponent implements OnInit {
       .then(balanceTeamsProjection => this.balancedTeams = balanceTeamsProjection,
         err => console.error('Cannot balance players', err));
   }
+
+  openAddPlayerDialog() {
+
+    const dialogRef = this.dialog.open(AddPlayerDialogViewComponent, {
+      width: '300px',
+      data: {groupId : this.groupId, currentPlayers: this.game.players}
+    });
+
+    dialogRef.afterClosed().subscribe((result: UserProjection[]) => {
+      if (result) {
+        const playersList: GamePlayersList = new GamePlayersList(result.map(value => value.id));
+        this.gameService.addPlayers(this.groupId, this.game.id, playersList).toPromise()
+          .then((newPlayers: UserProjection[]) => this.game.players = newPlayers,
```

Comment:
Just clarifying. Should we add new players to the list or replace with new ones?

Reply:
I prefer to use the response from the server with an up-to-date game players list. I assume someone else could be also adding players meanwhile, thus a user will have the latest version. What do you think?

Reply:
Totally agree, didn't get that gameService.addPlayers returns all merged players